otredis.go

```diff
@@ -2,34 +2,77 @@ package otredis
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
 	"github.com/go-redis/redis"
-	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
you might want to update your `goimports`:
https://github.com/golang/go/issues/28428
https://go-review.googlesource.com/c/tools/+/152000